### PR TITLE
When the patient label changes, we don't want to trigger an update to the carepoint data.

### PR DESCRIPTION
**Pull request by thehandcoder**

### CODE REVIEW

File: updates/update_patients_wc.php [hunk 1]
```diff
@@ -505,91 +506,24 @@ function wc_patient_updated(array $updated)
...
-                    'is_patient_match' => $is_patient_match
-                  ]
-            );
+            // Store this in the comment on the patient
+            $cpPatient              = $gpPatient->cpPat;
+            $gpComments             = $cpPatient->getGpComments();
+            $gpComments->first_name = $updated['first_name'];
+            $gpComments->last_name  = $updated['last_name'];
+            $gpComments->birth_date = $updated['birth_date'];
+            $cpPatient->setGpComments($gpComments);
```
Comment (Hobstweedl) [inline updates/update_patients_wc.php:526]: Why not eliminate the `setGpComments and just do
```
$cpPatient->cmt = $gpComments->toString();
$cpPatient->save();
```

File: updates/update_patients_wc.php [hunk 2]
```diff
@@ -221,7 +221,12 @@ function wc_patient_updated(array $updated)
         return null;
     }
 
-    if ($updated['patient_inactive'] !== $updated['old_patient_inactive']) {
+    // Since we've already been matched, we can grab the patient and use them
+
+    $gpPatient = GpPatient::where('patient_id_cp', $updated['patient_id_cp']);
```
Comment (Hobstweedl) [inline updates/update_patients_wc.php:226]: Need to `get()` or `first()`

File: updates/update_patients_wc.php (hunk 1 above)
Comment (thehandcoder) [reply updates/update_patients_wc.php:526]: I tend to make getters and setters when accessing and setting data from outside the object.  IMO it adds a little more integrity to the process by requiring that you pass an appropriate object.

File: updates/update_patients_wc.php (hunk 2 above)
Comment (thehandcoder) [reply updates/update_patients_wc.php:226]: I always forget about this.   I'm so used to my way of doing it that I even put 'new' in front of the class name

**Pull request merged**